### ISSUE TITLE
False positive in devel-dependancy

### ISSUE THREAD

**Issue by s-t-e-v-e-n-k** [rpm-software-management/rpmlint#1091]
An update to python-sortinghat has triggered:

```
[  119s] python310-sortinghat.noarch: E: devel-dependency python310-django-cors-headers
[  119s] python311-sortinghat.noarch: E: devel-dependency python311-django-cors-headers
[  119s] python39-sortinghat.noarch: E: devel-dependency python39-django-cors-headers
```

When django-cors-headers is expanded from python-django-cors-headers. Not really sure how best to quash this one.

**Comment by danigm** [comment]
The [TagsCheck looks for all dependencies](https://github.com/rpm-software-management/rpmlint/blob/main/rpmlint/checks/TagsCheck.py#L342) and the check done to confirm that's a devel package is a [regular expression](https://github.com/rpm-software-management/rpmlint/blob/main/rpmlint/checks/FilesCheck.py#L166) that in this case matches with the name.

So... with the current code, this package name (`python-django-cors-headers`) is problematic and anything that depends on it will throw this error.

For this specific case, it's possible to just filter this error, but I don't know if that's a okay solution to this problem or if this could be something that happens more times in the future.
